packages/tpetra/core/inout/mmio_Tpetra.h

```diff
@@ -0,0 +1,127 @@
+/* 
```

Comment:
@seheracer I couldn't find any info on what license governs this software. Any idea what it is?